### ISSUE TITLE
数据集

### ISSUE THREAD

**Issue by Aunnyxieyuzhou** [yt556677/ReFICR#4]
![image](https://github.com/user-attachments/assets/a184b65c-1eee-49b9-a8a0-6c1312ef825e)
作者你好，想问一下这个提供的数据是针对chujing007/ReFICR_inspired这个模型的吗，不是很明白数据集这一块是如何处理的，不知道有没有详细一点的说明呢

**Comment by zwang0621** [comment]
您好，我在复现过程中也遇到了一些问题，可以一起交流一下吗

**Comment by Aunnyxieyuzhou** [comment]
> 您好，我在复现过程中也遇到了一些问题，可以一起交流一下吗

好呀

**Comment by Aunnyxieyuzhou** [comment]
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
The `seen_tokens` attribute is deprecated and will be removed in v4.41. Use the `cache_position` model input instead.
在推理的时候遇到这个问题，但是倒也不影响运行，不知道这是什么原因呀

**Comment by zwang0621** [comment]
> > 您好，我在复现过程中也遇到了一些问题，可以一起交流一下吗
> 
> 好呀

可以加我一下vx吗，在这里讨论不太方便。
_Moonboy_

**Comment by zwang0621** [comment]
> > 您好，我在复现过程中也遇到了一些问题，可以一起交流一下吗
> 
> 好呀

可以加我一下vx吗，在这讨论不太方便。
        
__Moonboy__

**Comment by zwang0621** [comment]
> > 您好，我在复现过程中也遇到了一些问题，可以一起交流一下吗
> 
> 好呀

可以加我一下vx吗，在这讨论不太方便。
**825827981@qq.com**  前面的数字

**Comment by Xuer0817** [comment]
> ![image](https://github.com/user-attachments/assets/a184b65c-1eee-49b9-a8a0-6c1312ef825e) 作者你好，想问一下这个提供的数据是针对chujing007/ReFICR_inspired这个模型的吗，不是很明白数据集这一块是如何处理的，不知道有没有详细一点的说明呢

你好，你现在知道这个是怎么处理的了吗 @Aunnyxieyuzhou
